### ISSUE TITLE
Allow app to minimize to system tray where available

### ISSUE THREAD

**Issue by qdot** [intiface/intiface-central#43]
There's probably a flutter package for this.

**Comment by qwksilver** [comment]
I was about to make this as a feature request, is there a plan to implement this, it would be really nice.

**Comment by qdot** [comment]
Yup, I've at least pulled the package to do it in, mostly trying to figure out what the menu interface needs to look like.

**Comment by qdot** [comment]
Just tried this, all system tray packages for flutter seem to be broken right now. Windows just stalls on initializing the tray w/ system_tray, no errors, just freezes.

So this is basically icebox'd until there's a decent, reliable library to do this.

**Comment by qwksilver** [comment]
how about an option to minimize after launching in the meantime?
